Formula/b/backtrace_crashpad.rb

```diff
@@ -0,0 +1,77 @@
+    include.install "client/annotation_list.h" => "backtrace/crashpad/client/annotation_list.h"
+    include.install "client/settings.h" => "backtrace/crashpad/client/settings.h"
+    include.install "client/simple_string_dictionary.h" => "backtrace/crashpad/client/simple_string_dictionary.h"
+    include.install "client/length_delimited_ring_buffer.h" => "backtrace/crashpad/client/length_delimited_ring_buffer.h"
+    include.install "client/crashpad_info.h" => "backtrace/crashpad/client/crashpad_info.h"
+    include.install "client/prune_crash_reports.h" => "backtrace/crashpad/client/prune_crash_reports.h"
+    include.install "client/simulate_crash_mac.h" => "backtrace/crashpad/client/simulate_crash_mac.h"
+    include.install "client/crash_report_database.h" => "backtrace/crashpad/client/crash_report_database.h"
+    include.install "client/client_argv_handling.h" => "backtrace/crashpad/client/client_argv_handling.h"
+    include.install "client/annotation.h" => "backtrace/crashpad/client/annotation.h"
+
+    mkdir "#{include}/backtrace/util/file"
+    include.install "util/file/file_io.h" => "backtrace/util/file/file_io.h"
+    mkdir "#{include}/backtrace/util/misc"
+    include.install "util/misc/capture_context.h" => "backtrace/util/misc/capture_context.h"
+    mkdir "#{include}/backtrace/base"
+    include.install "third_party/mini_chromium/mini_chromium/base/scoped_generic.h" => "backtrace/base/scoped_generic.h"
+    mkdir "#{include}/backtrace/base/mac"
+    include.install "third_party/mini_chromium/mini_chromium/base/mac/scoped_mach_port.h" => "backtrace/base/mac/scoped_mach_port.h"
+    mkdir "#{include}/backtrace/base/files"
+    include.install "third_party/mini_chromium/mini_chromium/base/files/file_path.h" => "backtrace/base/files/file_path.h"
+    include.install "third_party/mini_chromium/mini_chromium/base/files/scoped_file.h" => "backtrace/base/files/scoped_file.h"
+    mkdir "#{include}/backtrace/build"
+    include.install "third_party/mini_chromium/mini_chromium/build/build_config.h" => "backtrace/build/build_config.h"
+    include.install "third_party/mini_chromium/mini_chromium/build/buildflag.h" => "backtrace/build/buildflag.h"
+
+    cmake_config = <<-EOF
+add_library(backtrace::crashpad STATIC IMPORTED)
+set_target_properties(backtrace::crashpad PROPERTIES
+  IMPORTED_LOCATION "${CMAKE_CURRENT_LIST_DIR}/../../libbacktrace_crashpad_client.a")
+target_include_directories(backtrace::crashpad 
+  INTERFACE "${CMAKE_CURRENT_LIST_DIR}/../../../include/backtrace")
+set(BacktraceCrashpad_FOUND TRUE)
+    EOF
+    cmake_version_config = <<-EOF
+set(PACKAGE_VERSION "#{version}")
+if(PACKAGE_FIND_VERSION VERSION_EQUAL PACKAGE_VERSION)
+  set(PACKAGE_VERSION_EXACT TRUE)
+endif()
+if(NOT PACKAGE_FIND_VERSION VERSION_GREATER PACKAGE_VERSION)
+  set(PACKAGE_VERSION_COMPATIBLE TRUE)
+else(NOT PACKAGE_FIND_VERSION VERSION_GREATER PACKAGE_VERSION)
+  set(PACKAGE_VERSION_UNSUITABLE TRUE)
+endif()
+    EOF
```

Comment:
Homebrew formulae should not ship their own configuration. This should either come from upstream or not at all

Reply:
Ah, interesting. @rick-bt @gm4sl Would you take a PR ti generate CMake configs?

Reply:
Hi @jviotti - yes, we would welcome that.